model/feature_flags.go

```diff
@@ -109,6 +111,7 @@ func (f *FeatureFlags) SetDefaults() {
 	f.AnnualSubscription = false
 	f.ReduceOnBoardingTaskList = false
 	f.ThreadsEverywhere = false
+	f.GlobalDrafts = false
```

Comment:
@mylonsuren can you help submit a PR to set this default true for v7.6?

Reply:
I think it should be submitted against master and then @amyblais will cherry pick to v7.6

Reply:
Ticket: https://mattermost.atlassian.net/browse/MM-48622.